src/main/java/graphql/annotations/connection/AbstractPaginatedData.java

```diff
@@ -18,9 +18,9 @@
 
 public abstract class AbstractPaginatedData<T> implements PaginatedData<T> {
 
-     boolean hasPreviousPage;
-     boolean hasNextPage;
-     Iterable<T> data;
+    Iterable<T> data;
```

Comment:
should not be private also?

Reply:
I don't think so, because the extending class may use the data (not like the pages, where there is no need to use them once you know if they are true or false)

Reply:
so set it to protected

Reply:
done